docs/products/kafka/howto/terraform-governance-approvals.md

```diff
@@ -95,35 +95,12 @@ be integrated to validate changes and enforce compliance.
 ### Step 2. Map GitHub users to Aiven identities
 
 To verify requesters and approvers, assign them to the appropriate user group
```

Comment:
This is not something I'm super familiar with, but should this instead say something about the user ID mapping instead of groups since it's now only the external_identity resource?
```suggestion
To verify requesters and approvers, map their GitHub user IDs to their Aiven user IDs
```

Reply:
Good point and makes sense. I will update this.